MVCTask/MVCTask/Controllers/NewUserController.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using Castle.Components.DictionaryAdapter;
+using Microsoft.Ajax.Utilities;
+using MVCTask.Core.Interface;
+using MVCTask.Models;
+
+namespace MVCTask.Controllers
+{
+    public class NewUserController : BaseController
+    {
+        private readonly ITestServise _servise;
+        
+        public NewUserController(IActionInvoker actionInvoker,ITestServise servise)
+        {
+            if (actionInvoker==null||servise==null)
+            {
+                throw new ArgumentNullException("Some OF");
+            }
+            ActionInvoker = actionInvoker;
+            _servise = servise;
+        }
+        // GET: NewUser
+        public ActionResult NewUser()
```

Comment:
please add parameter userId to action and remove parsing of route data - it's looks like bicycle